From 1107222755077a510b82331c149610a88585e046 Mon Sep 17 00:00:00 2001
From: Terry <t2tritton@gmail.com>
Date: Fri, 10 Sep 2021 17:00:22 +0800
Subject: [PATCH] add default host ship and password as we don't have a
 keyboard

---
 snis_client.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/snis_client.c b/snis_client.c
index 7125659e..8377a335 100644
--- a/snis_client.c
+++ b/snis_client.c
@@ -20463,7 +20463,11 @@ static void init_net_setup_ui(void)
 	char *preferred_shipname = snis_prefs_read_default_ship_name(xdg_base_ctx);
 
 	memset(net_setup_ui.lobbyname, 0, sizeof(net_setup_ui.lobbyname));
-	strcpy(net_setup_ui.lobbyname, "localhost");
+	strcpy(net_setup_ui.lobbyname, "10.10.4.105");
+	memset(net_setup_ui.shipname, 0, sizeof(net_setup_ui.shipname));
+	strcpy(net_setup_ui.shipname, "Bob");
+	memset(net_setup_ui.password, 0, sizeof(net_setup_ui.password));
+	strcpy(net_setup_ui.password, "fish");
 	strcpy(net_setup_ui.solarsystem, "");
 	y += yinc;
 	net_setup_ui.lobbyservername =
